.env

```
@@ -0,0 +1,35 @@
+# 서버
+SERVER_PORT=8080
+BASE_URL=http://localhost:8080
+
+# 데이터베이스
+DB_URL=jdbc:mysql://localhost:3306/pventure
+DB_USERNAME=root
+DB_PASSWORD=1234
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_URL key

(UnorderedKey)